Formula/fizz.rb

```diff
@@ -35,8 +35,11 @@ class Fizz < Formula
   fails_with gcc: "5"
 
   def install
+    args = []
+    args << "-DLIBRT_LIBRARY=/usr/lib/x86_64-linux-gnu/librt.so" if OS.linux?
```

Comment:
CC @Homebrew/linux in case this is not a good idea.

Reply:
`/usr/lib/x86_64-linux-gnu/librt.so -> /lib/x86_64-linux-gnu/librt.so.1`

It's a system library so it's fine. It's a shame though that it can't find it by itself in the standard linux lib path.

Reply:
Normally I would say we shouldn't do this becuase `librt` is part of `glibc`, and hardcoding it like this would break for users with brewed `glibc` or who are running on distros which don't use a platform subdirectory like `x86_64-linux-gnu` the way Ubuntu does.  However, this bottle is relocatable so nobody should need to build it from source outside of CI.  Having said this, it is still very odd that this is even needed at all.

Reply:
> hardcoding it like this would break for users with brewed `glibc` or who are running on distros which don't use a platform subdirectory like `x86_64-linux-gnu` the way Ubuntu does. 

This is what I was concerned about.

> Having said this, it is still very odd that this is even needed at all.

Agreed. Not sure why their `FindLibRT.cmake` script fails to find `librt`, given it just uses CMake's `find_library` function. Trying to call that function on its own in the Homebrew Docker container seems to find `librt` just fine.